### PR TITLE
CreateWorkout: add delete exercises/sets functionality

### PR DESCRIPTION
**Pull request by klymenkoo**
del

### CODE REVIEW

File: components/ExerciseCard.js [hunk 1]
```diff
@@ -52,14 +50,36 @@ class ExerciseCard extends React.Component {
     this.props.onDescChange(newDesc, id);
   }
 
-  updateSet(id, newSet){
+  deleteSet(id) {
+    console.warn(id);
```
Comment (AlexeiGitH) [inline components/ExerciseCard.js]: Was it for debugging? 😬

**Comment by klymenkoo** [comment]
@AlexeiGitH fixed

File: components/ExerciseCard.js [hunk 2]
```diff
@@ -71,21 +91,37 @@ class ExerciseCard extends React.Component {
...
+                { ...FontStyles.h1, ...FontStyles.bold }
+              ]}
+              placeholder="Exercise name"
+              onChangeText={this.changeName}
+              underlineColorAndroid="transparent"
+            >
+              {this.props.exercise.name}
+            </TextInput>
+            <Icon
+              name="minus-circle"
```
Comment (AlexeiGitH) [inline components/ExerciseCard.js]: Not loving the icon, it stands out too much. Could you replace it with **[close](https://material.io/tools/icons/?search=close&icon=close&style=baseline)** icon?

Comment (klymenkoo) [reply components/ExerciseCard.js]: @AlexeiGitH i agree. Fixed. It looks better now.

**Pull request merged**